### ISSUE TITLE
Space after comma followed by template string fails

### ISSUE THREAD

**Issue by ash2k** [babel/babel-eslint#156]
The following snippet fails with `A space is required after ','  comma-spacing`:

```javascript
x(1, `template literal string`);

function x() {}
```
`.eslintrc`:
```
parser: babel-eslint
environment:
  node: true
  mocha: true
ecmaFeatures:
  generators: true
  modules: true
  blockBindings: true
  arrowFunctions: true
  destructuring: true
  forOf: true
  templateStrings: true
```
Babel v5.8.9
eslint v0.24.1
babel-eslint v4.0.3

**Comment by sebmck** [comment]
Fixed as of 4.0.4/3.1.30.

**Comment by ash2k** [comment]
Works for me, thanks a lot!
